gvsbuild/utils/builder.py

```diff
@@ -586,6 +586,28 @@ def build(self, projects):
 
         log.close()
 
+    def __make_folder_reproducible(self, parent):
+        """Iterate over DLL and EXE files inside of 'parent' and make them
+        reproducible with the help of ducible.
+
+        'parent' needs to be an absolute path
+        """
+        proj = self.__project
+
+        for file in os.listdir(parent):
+            path = os.path.join(proj.pkg_dir, file)
+            if os.path.isdir(path):
+                self.__make_folder_reproducible(path)
+            else:
+                if file.endswith(".exe") or file.endswith(".dll"):
+                    noextension_name = file[:-4]
+                    pdb_name = os.path.join(parent, noextension_name, ".pdb")
```

Comment:
Just a passing suggestion that you could simplify things here with pathlib and use subprocess

```python
from pathlib import Path
import subprocess

paths = Path(parent).glob('**/*') 
files = [p for p in paths if p.is_file() and p.suffix in ['.exe', '.dll']]
for file in files:
    pdb = file.with_suffix('.pdb')
    if not pdb.exists():
        pdb = ''
    image = Path(self.__project.pkg_dir) / file
    subprocess.run(f"ducible {image} {pdb}", shell=True)
```

Reply:
Aww, thanks!

Reply:
Mergedd!! <3